metadata.yaml

```diff
@@ -38,7 +38,7 @@ authors:
 copyright: "2022, Association of Universities for Research in Astronomy, Inc. (AURA)"
 
 # Description. A short, 1-2 sentence statemement used by document indices.
-description: "The First-Look Analysis and Feedback Functionality (FAFF) working group, identified the need for a service that is capable of executing user-defined operations/computation following user-defined rules. This service has been referred  to as Catcher. In essence, the Catcher is not that different from the Watcher or other event handling off-the-shelf technologies like Kapacitor. This technote explores the idea of using influx2.0 event processing capabilities to as an alternative implementation for the Catcher."
+description: "This technote contains design considerations for the Catcher, an event-driven processing component for the Rubin Observatory Control System. We briefly review the existing functionality of the system and discuss the use case for the Catcher. The idea originated from the First-Look Analysis and Feedback Functionality (FAFF) working group, and a more in-depth analysis of the use cases can be found in their reports."
```

Comment:
semantic line feeds should be added here...

Reply:
Uhm.. not sure how to do that in a yaml file.. I'll try the trick we use in the script and csc configs..